util/weekly_build/apptests/test_ufswm.sh

```diff
@@ -3,3 +3,18 @@
 # Do some stuff, and exit non-zero if UFSWM cannot be successfully built and
 # tested (ideally, have a discernably different error condition if there are
 # numerical differences)
+
+cd ${RUNDIR}
+
+# The following need to be cleaned up for proper variable substitution; this is a one-off test
+BRANCH=spack-stack-automation
+
+git clone -b ${BRANCH} --single-branch --recurse-submodules https://github.com/rickgrubin-noaa/ufs-weather-model.git
+cd ufs-weather-model/tests
+
+# -r ==> rocoto ; not strictly necessary, can run without
+./rt.sh -a epic -k -r -n "control_c48 intel"
```

Comment:
This looks great-- Is there a way to generalize this for systems that won't use the "epic" job scheduler account? Maybe a variable that could be set in the site configs?

Reply:
Indeed, generalizations are necessary for anything that may be different across various `tier1` and `tier2` hosts. 

Certainly 

- `branch`
- `job scheduler`
- `job scheduler account` 

are the obvious variables; `rt.sh` has numerous options that could be parameterized in this sense, and should be. If that's preferable prior to merging, that is fine / I'll do that.

Reply:
Scripts cleaned up:

- `06_AppTests.sh`
  - intended for site-specific modification with respect to how `ufs-weather-model/tests/rt.sh` is executed with respect to what arguments are passed to that overarching test script
- `apptests/test_ufswm.sh`
  - `ufs-weather-model/tests/rt.sh` will parse arguments passed through from `06_AppTests.sh`

Tested on **orion** via `Jenkins` and from the command line.

Changes to `ufs-weather-model` are required to test against a stack built via automation, as `ufs-weather-model/tests/rt.sh` expects to use a most recently released / published stack:

- `ufs-weather-model/tests/rt.sh`
  - may need to change the value of `dprefix` to a location where the `$USER` running the automated build has `rwx` privileges
- `ufs-weather-module/modulefiles/ufs_<host>.<compiler>.lua` must be changed to reflect:
  - location of automated build stack modules
  - version(s) of modules loaded, if different from past stack releases
    - e.g. newer compiler